scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -200,3 +201,57 @@ def test_from_dcm_ortho_output():
         eye3d[:, i, i] = 1.0
 
     assert_array_almost_equal(mult_result, eye3d)
+
+
+def test_from_1d_single_rotvec():
+    rotvec = [1, 0, 0]
+    expected_quat = np.array([0.4794255, 0, 0, 0.8775826])
+    result = Rotation.from_rotvec(rotvec)
+    assert_array_almost_equal(result.as_quaternion(), expected_quat)
+
+
+def test_from_2d_single_rotvec():
+    rotvec = [[1, 0, 0]]
+    expected_quat = np.array([[0.4794255, 0, 0, 0.8775826]])
+    result = Rotation.from_rotvec(rotvec)
+    assert_array_almost_equal(result.as_quaternion(), expected_quat)
+
+
+def test_from_generic_rotvec():
+    rotvec = [
+            [1, 2, 2],
+            [1, -1, 0.5],
+            [0, 0, 0]
+            ]
+    expected_quat = np.array([
+        [0.3324983, 0.6649967, 0.6649967, 0.0707372],
+        [0.4544258, -0.4544258, 0.2272129, 0.7316889],
+        [0, 0, 0, 1]
+        ])
+    assert_array_almost_equal(
+            Rotation.from_rotvec(rotvec).as_quaternion(),
+            expected_quat)
+
+
+def test_from_rotvec_small_angle():
+    # Small theta
+    rotvec = np.array([1, 0.00001, 0.00001]) * 5e-6
```

Comment:
The axis is not generic enough (along `x` practically) and I don't understand why you defined it this way. Could you test with a more interesting axis and perhaps somewhat lager angle?

Also we probably want to test on a stack of vectors where large and small angles are present.